tests/test_dummy.py

```diff
@@ -0,0 +1,2 @@
+def test_dummy_test():
+    assert not 1 == 2
```

Comment:
I think it's better to have a positive check, like
```suggestion
    assert 1 == 1
```

Since it's just a dummy test I think that's ok